cli/tests/fetch_data/README.md

```diff
@@ -0,0 +1,7 @@
+This directory and subdirectories contains data for some of the fetch
+benchmarks.
+
+The following list of URLs is intended to credit the original
+creators/distributors of these files:
+
+- https://corpus.canterbury.ac.nz/descriptions/
```

Comment:
Not super happy about adding fixture files to repo, maybe you could get away without it, by creating some data on the fly in appropriate request handlers in `test_server`? We don't really need to have structured data, it's just a matter of downloading proper number of bytes.

Reply:
@bartlomieju I can change the benchmarks to use mock data instead, that does make for a cleaner codebase. 

Wondering if it should still be written to disk, or just stored/served from in memory? Or maybe both? Just want to make sure the benchmarks is capturing the right scenarios.

Reply:
@ACov96 I think generating data on demand and in-memory only should be sufficient for this benchmark.